src/components/plots/errorBarPlot.ts

```diff
@@ -0,0 +1,47 @@
+module Plottable {
+export module Plot {
+  export class ErrorBar<X> extends AbstractXYPlot<X,number> {
```

Comment:
That's a release note/example thing right? Just following precedence from other plots